### PR TITLE
[UX] Add some copy in the Wine manager explaining what are the different Wines in linux

### PR DESCRIPTION
**Pull request by arielj**
This PR addressed a common problem that users try `Wine-GE-Proton-LoL` without knowing it will most likely fail if not used for League of Legends.

This can also help address common questions:
- which wine variant is recommended (based on Glorious Eggroll's reddit posts)
- why we allow downloading proton-ge if wine-ge is recommended
- hopefully users understand the part about Proton-GE-Proton providing useless logs too

image
image
image

One extra note: since I had to add some style and the scss file simple enough, I moved to be a plain css file since we can now use native css nesting. We can eventually stop using sass.

---

Use the following Checklist if you have changed something on the Backend or Frontend:

- [ ] Tested the feature and it's working on a current and clean install.
- [ ] Tested the main App features and they are still working on a current and clean install. (Login, Install, Play, Uninstall, Move games, etc.)
- [ ] Created / Updated Tests (If necessary)
- [ ] Created / Updated documentation (If necessary)

### CODE REVIEW

File: src/frontend/screens/WineManager/index.tsx [hunk 1]
```diff
@@ -110,6 +114,43 @@ export default function WineManager(): JSX.Element | null {
     }
   }, [repository])
 
+  let wineVersionExplanation = <></>
```
Comment (CommandMC) [inline src/frontend/screens/WineManager/index.tsx:117]: The `useMemo` hook should be used for values like this, e.g.
```ts
const wineVersionExplanation = useMemo(() => { /* switch case in here, returning the appropriate value */ }, [repository.type])
```

That'll prevent the value from being re-calculated on every re-render

File: src/frontend/screens/WineManager/index.css
Comment (CommandMC) [inline src/frontend/screens/WineManager/index.css:1]: Ideally use `git mv` when renaming a file (even when making changes to it). That'll keep its history alive (right now, it looks like this whole file's contents were added by this PR, which isn't the case) and also makes it easier to review what's actually relevant

Comment (arielj) [reply src/frontend/screens/WineManager/index.css:1]: I just tried `git mv` and the thing is that this PR does update the whole file (I added the `.wineManager` class wrapper to nest elements inside, so all lines have an updated indentation)

File: src/frontend/screens/WineManager/index.css [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    }
+  }
+
+  .infoBox {
+    width: 80%;
+    margin: 0 auto;
+    & svg {
+      margin-inline-end: 0.5rem;
+    }
+  }
```
Comment (arielj) [inline src/frontend/screens/WineManager/index.css:54]: this `.infoBox` is the only new thing in this file, the rest is just updated indentation (I added a `.wineManager` wrapper selector)

File: src/frontend/screens/WineManager/index.css
Comment (CommandMC) [reply src/frontend/screens/WineManager/index.css:1]: Hmm, usually GitHub picks up on files renamed with `git mv` (it should show a "Renamed from ..." note somewhere), and telling it to then not show whitespace changes should only show the new parts of the file

But oh well, this isn't a huge issue

**Review by flavioislima — APPROVED** [review]
lgtm

**Pull request merged**